src/tool/mod.rs

```diff
@@ -64,9 +68,27 @@ impl Tools {
             mkfat: Tool::find("mkfs.fat", dryrun).map_err(|_| {
                 anyhow!("mkfs.fat is required for creating FAT filesystems. Please install the 'dosfstools' package.")
             })?,
+            // TODO: Technically don't need ext4 if only using btrfs
             mkext4: Tool::find("mkfs.ext4", dryrun).map_err(|_| {
                 anyhow!("mkfs.ext4 is required for creating ext4 filesystems. Please install the 'e2fsprogs' package.")
             })?,
+            mkbtrfs: if is_btrfs {
+                Some(Tool::find("mkfs.btrfs", dryrun).map_err(|_| {
+                anyhow!("mkfs.btrfs is required for creating btrfs filesystems. Please install the 'btrfs-progs' package.")
+            })?)
+            } else {
+                None
+            },
+            btrfs: if is_btrfs {
+                Some(Tool::find("btrfs", dryrun).map_err(|_| {
+                anyhow!("btrfs is required for creating btrfs filesystems. Please install the 'btrfs-progs' package.")
+            })?)
+            } else {
+                None
+            },
+            git: Tool::find("git", dryrun).map_err(|_| {
+                anyhow!("git is required for using ALMA. Please install the 'git' package.")
```

Comment:
when is it used?

Reply:
If the presets are in a git repo, and to install Omarchy.